### PR TITLE
Assignment review issue

### PR DESCRIPTION
**Pull request by Ranipandit**

### CODE REVIEW

File: src/components/courses-list/index.js [hunk 1]
```diff
@@ -1,238 +1,243 @@
...
+  rootLoader: {
+    display: "flex",
+    flexGrow: 1,
+    justifyContent: "center",
+    marginTop: "46vh"
+  },
+  root: {
+    display: "flex",
+    justifyContent: "center",
+    paddingBottom: 50
```
Comment (amarkrsinha1997) [inline src/components/courses-list/index.js]: theme.spacing.unit

Comment (amarkrsinha1997) [inline src/components/courses-list/index.js]: bad practice use theme.spacing.unit

File: src/components/courses-list/index.js [hunk 2]
```diff
@@ -1,238 +1,243 @@
...
+  },
+  rootContent: {
+    paddingTop: theme.spacing.unit * 5,
+    display: "flex",
+    flexDirection: "column",
+    width: "80%",
+    [theme.breakpoints.down("sm")]: {
+      width: "100%",
+      paddingLeft: 10,
```
Comment (amarkrsinha1997) [inline src/components/courses-list/index.js]: theme.spacing.unit

File: src/components/courses-list/index.js [hunk 3]
```diff
@@ -1,238 +1,243 @@
...
+      paddingRight: 10
+    }
+  },
+  courseEditButton: {
+    display: "block",
+    float: "right",
+    margin: "0",
+    top: "auto",
+    float: "right",
```
Comment (amarkrsinha1997) [inline src/components/courses-list/index.js]: float:"right" is twice

**Comment by amarkrsinha1997** [comment]
the image still needs to be more responsive can you do that it's smaller than it's actual width @Ranipandit.
Finish it up quickly so we can merge it and move on.

File: src/components/courses-list/index.js (hunk 1 above)
Comment (Khushboopandit) [reply src/components/courses-list/index.js]: resolve

File: src/components/courses-list/index.js (hunk 2 above)
Comment (Khushboopandit) [reply src/components/courses-list/index.js]: resolve

File: src/components/courses-list/index.js (hunk 3 above)
Comment (Khushboopandit) [reply src/components/courses-list/index.js]: resolve

**Comment by amarkrsinha1997** [comment]
You can go ahead with this branch @abhishekgupta92

**Pull request merged**